tests/smoke-maven-group-multidir.yaml

```diff
@@ -212,11 +212,11 @@ output:
                       groups: []
                       metadata:
                         packaging_type: pom
-                      requirement: 1.5.12.RELEASE
+                      requirement: 1.5.11.RELEASE
```

Comment:
It is not something big since the pattern same, just wanted to check if there is a reason we didn't use the same version `1.5.12.RELEASE`